src/services/balances.rs

```diff
@@ -45,6 +46,14 @@ pub fn balances(
         })
         .collect();
 
+    service_balances.sort_by(|a, b| {
+        a.fiat_balance
+            .parse::<f64>()
```

Comment:
Any issues here, that value might be to large or have too many decimals?

Reply:
I would have to add unit tests for this parse method. As we use it in this same file too in line 44.

Reply:
I see we fallback to 0 ... so doesn't matter for now :)

Reply:
Not sure what the alternative would be however.

Reply:
Should we introduce something like this: https://docs.rs/rug/0.6.0/rug/ ?